components/gitpod-cli/cmd/validate.go

```diff
@@ -89,14 +88,13 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 	var dockerContext string
 	switch img := gitpodConfig.Image.(type) {
 	case nil:
-		// TODO: default image won't be always gitpod/workspace-full, just added for compatibility before server is deployed
-		image = "gitpod/workspace-full:latest"
-		if defaultImage, err := getDefaultWorkspaceImage(ctx); err != nil {
-			fmt.Printf("Failed to get default workspace image: %v\n", err)
-		} else {
-			fmt.Println("Using default workspace image:", defaultImage)
-			image = defaultImage
+
+		image = wsInfo.DefaultWorkspaceImage
+		if image == "" {
```

Comment:
@mustard-mh it should not be necessary, it should be part of supervisor already?

Reply:
the same with `gp init`

Reply:
Addressed https://github.com/gitpod-io/gitpod/pull/18723/commits/1941376e5d72b8436bf1c6ee91057df1df1b889e